shaders/gshader_to_mshader/slang/gshader_to_mshader.geom.slang

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2023-2025, Mobica Limited
+ * Copyright (c) 2025, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+struct VSOutput
+{
+    float4 Pos : SV_Position;
+    float3 Normal;
+};
+
+struct GSOutput
+{
+    float4 Pos : SV_POSITION;
```

Comment:
As you're mixing `SV_Position` and `SV_POSITION` here: I think the "correct" spelling would be `SV_Position`, even though probably all compilers accept `SV_POSITION` as well.